hack/kustomization/whereabouts/kustomization.yaml

```diff
@@ -7,5 +7,16 @@ resources:
 - ./whereabouts.cni.cncf.io_ippools.yaml
 - ./whereabouts.cni.cncf.io_overlappingrangeipreservations.yaml
 
-transformers:
-  - patch-ip-reconciler-job.yaml
+patches:
```

Comment:
Thanks for the changes
Can we do it please with this kustomization style ?
https://github.com/kubevirt/kubevirtci/blob/main/cluster-up/cluster/kind-1.22-sriov/sriov-components/manifests/kustomization.yaml#L9

and then the sed in the script is not needed

Reply:
`newTag` i think ?

something like this
```
apiVersion: kustomize.config.k8s.io/v1beta1
kind: Kustomization
resources:
- grafana-deployment.yaml
images:
  - name: grafana/grafana
    newName: quay.io/kubevirtci/grafana-grafana
    newTag: <TAG>
```

Reply:
And.. how do you set `<TAG>`? :)

Reply:
envsubst / sed it according "whereabouts_version", wdyt?

Reply:
Done, PTAL!